cmd/launcher/main.go

```diff
@@ -96,10 +98,19 @@ func main() {
 
 	// recreate the logger with  the appropriate level.
 	logger = logutil.NewServerLogger(opts.Debug)
+	var multiSlogger *multislogger.MultiSlogger
```

Comment:
Why not `multiSlogger := multislogger.New()` here?

Reply:
fixed